cdap-docs/developer-manual/source/building-blocks/mapreduce-programs.rst

```diff
@@ -376,14 +376,7 @@ MapReduce and Resources
 =======================
 
 Both the YARN container size and the number of virtual cores used in a MapReduce job can be specified
-as part of the configuration. They can also be set at runtime through the use of runtime arguments. An
-example of this is shown in the :ref:`Purchase <examples-purchase>` example, where the memory requirements
-are set:
-
-.. literalinclude:: /../../../cdap-examples/Purchase/src/main/java/io/cdap/cdap/examples/purchase/PurchaseHistoryBuilder.java
-   :language: java
-   :lines: 47-57
-   :append: ...
+as part of the configuration. They can also be set at runtime through the use of runtime arguments.
```

Comment:
should this be replaced by a literal piece of code, instead of deleting it? Otherwise, it is not clear what configurations are required.